packages/cli/src/Ldap/helpers.ts

```diff
@@ -195,6 +195,7 @@ export const updateLdapConfig = async (config: LdapConfig): Promise<void> => {
 	}
 
 	await Db.collections.FeatureConfig.update({ name: LDAP_FEATURE_NAME }, { data: config });
+	LdapManager.updateConfig(config);
```

Comment:
shouldn't this have been before we encrypt the `bindingAdminPassword`?
I think this change might have broken the syncing function.
We can fix it tomorrow morning.